src/ReactSelectMaterialUi.tsx

```diff
@@ -95,6 +95,15 @@ class ReactSelectMaterialUi extends React.Component<ReactSelectMaterialUiProps,
 		);
 	}
 
+	public componentWillReceiveProps(props: ReactSelectMaterialUiProps) {
```

Comment:
As componentWillReceiveProps is deprecated, I cannot accept code changes using it.
[https://reactjs.org/docs/react-component.html#updating](url)